vscode-html/out/customData.js

```diff
@@ -0,0 +1,27 @@
+"use strict";

```

Comment:
It seems like those compiled files have Windows line-endings. I do think it could cause problems and should be changed to Unix. :)

Reply:
Ah, yes I compiled it on Windows. Fortunately I find a flag to specify the line ending now. :+1: 
https://github.com/microsoft/TypeScript/pull/2921